mypy/config_parser.py

```diff
@@ -112,6 +112,10 @@ def parse_config_file(options: Options, set_strict_flags: Callable[[], None],
         if not os.path.exists(config_file):
             continue
         try:
+            if 'MYPY_CONFIG_FILE_DIR' not in os.environ:
+                os.environ['MYPY_CONFIG_FILE_DIR'] = config_file
```

Comment:
Why are you setting the _file_ name as a directory? Shouldn't you use something like `os.path.dirname()`?

Reply:
Yes you are right. Thanks for noticing. I will change it in next commit.

Reply:
done